docs/dev_guide/contents/ci.rst

```diff
@@ -0,0 +1,40 @@
+.. _ci_jobs:
+
+***************************************
+What runs on our Continuous Integration
+***************************************
+
+Overall the aim of the Continuous Integration is to provide a way for developers to test their code on multiple platforms and help in the development process.
+While we can run many jobs, we aim to limit the selection depending in the context to ensure that the time taken is not wasted or ends up delaying a pull request or release.
+
+Currently we have several stages of CI jobs, some run on a pull request and some will only run on a schedule.
+
+1. "core" - Pull Request, Scheduled and Release
+   This runs a basic offline test suite on Linux for the latest version of Python we support.
+   It ensures that basic functionality works and that we don't have any regressions.
```

Comment:
For spaces to fix formatting https://sunpy--6037.org.readthedocs.build/en/6037/dev_guide/contents/ci.html
```suggestion
    This runs a basic offline test suite on Linux for the latest version of Python we support.
    It ensures that basic functionality works and that we don't have any regressions.
```

Reply:
This is what I get for not running it locally!